test/e2e/replica_set_readiness_probe/replica_set_readiness_probe_test.go

```diff
@@ -31,7 +32,7 @@ func TestReplicaSetReadinessProbeScaling(t *testing.T) {
 	t.Run("Stateful Set Reaches Ready State", mongodbtests.StatefulSetIsReady(&mdb))
 	t.Run("MongoDB is reachable", mongodbtests.IsReachableDuring(&mdb, time.Second*10,
 		func() {
-			t.Run("Delete Random Pod", mongodbtests.DeletePod(&mdb, rand.Intn(mdb.Spec.Members-1)))
+			t.Run("Delete Random Pod", mongodbtests.DeletePod(&mdb, rand.Intn(mdb.Spec.Members)))
```

Comment:
from the docs

`Intn returns, as an int, a non-negative pseudo-random number in [0,n) from the default Source. It panics if n <= 0.`

That means we could access `pod-3` which doesn't exist!

Reply:
Maybe I am missing something, but if `n=3`, `IntN` will return a number in the range `[0,3)`, so 3 will be excluded (notice the round parentheses as opposed to the square one).

Or am I missing something?

Reply:
nope you are absolutely right! I missed it 👍